CHANGELOG.md

```diff
@@ -9,14 +9,17 @@ Changes since last non-beta release.
 
 _Please add entries here for your pull requests that are not yet released._
 
-### Breaking Changes
-- Remove support & testing for Webpacker 3/4.
-- Add support & testing for Shakapacker v6
-- Requires separate compilations for server & client bundles (see [Webpack config](https://github.com/reactjs/react-rails/tree/master/test/dummy/config/webpack))
-- Replaces WebpackManifestContainer, which searched for assets in the webpack manifest, with SeparateServerBundleContainer, which expects a single server bundle file & does not use the webpack manifest at all.
-- All in one PR! #1274
-
+## Breaking Changes
+### Removed
+- Support & testing for Webpacker & Shakapacker v6. #1274
+- WebpackManifestContainer, which searched for assets in the webpack manifest. #1274
```

Comment:
Yes, Shakapacker v7 is currently required because we have code that calls `Shakapacker.config` 

If you want, we could change the constant back to `Webpacker` for backwards compatibility.